### PR TITLE
updated terms

### PR DESCRIPTION
**Pull request by Degoot-AM**

### CODE REVIEW

File: learners/reference.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ title: 'Glossary of Terms: Epiverse-TRACE'
 ## A
 
 [Airborne transmission]{#airborne}
-: Individuals become infected via contact with infectious particles in the air. Examples include influenza, COVID-19. [More information on airborne transmission](https://www.ncbi.nlm.nih.gov/books/NBK531468/).
+: Individuals become infected via contact with infectious particles in the air. Examples include influenza, and COVID-19. Click[here](https://www.ncbi.nlm.nih.gov/books/NBK531468/) for more information on airborne transmission.
```
Comment (avallecam) [inline learners/reference.md:8]:
```suggestion
: Individuals become infected via contact with infectious particles in the air. Examples include influenza and COVID-19. Several [causes and factors](https://www.ncbi.nlm.nih.gov/books/NBK531468/) influence airborne transmission.
```

**Review by avallecam — CHANGES_REQUESTED** [review]
Thank you for your edit suggestion. I requested one change for this related to #83

**Comment by avallecam** [comment]
@Degoot-AM I think this is a relevant edit because this is improving the writing. Do you agree if we reopen this PR to resolve the conversation and merge?

**Comment by Degoot-AM** [comment]
Yes, I agree.

File: learners/reference.md (hunk 1 above)
Comment (avallecam) [reply learners/reference.md:8]: @Degoot-AM if you agree you can also add this as commit to merge this :)

Comment (Degoot-AM) [reply learners/reference.md:8]: Yes, I do agree

**Review by avallecam — APPROVED** [review]
agreement on edits!

**Pull request merged**